build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     addRepos(repositories)
     dependencies {
         classpath 'com.android.tools.build:gradle:3.4.2'
-        classpath "org.mozilla.telemetry:glean-gradle-plugin:$versions.android_components"
+        classpath "org.mozilla.telemetry:glean-gradle-plugin:$versions.telemetry"
```

Comment:
@badboy if we're considering doing a major version bump, maybe we could as well also make this repo use the Gradle plugin as exported by a-c?

/cc @mdboom

Reply:
Yes -- we definitely should do that.  There is still an [issue blocking that](https://github.com/mozilla-mobile/android-components/pull/5476).

My current plan is to just file PRs with all of our consumers once this is working, so I wouldn't worry about this just yet.